samples/net/sockets/echo_client/prj.conf

```diff
@@ -0,0 +1,52 @@
+# Generic networking options
```

Comment:
Perhaps it would be good time to have common settings in `prj.conf` file and board specific stuff (if there is any) in `boards/${BOARD}.conf` file. One example of this is in `samples/net/rpl_node` sample. The original echo-client/server config files are a big mess and I would not like to introduce similar kind of setup here.

Reply:
+1 for that, I'll make IPv4/IPv6 and TCP/UDP configuration board-specific for starters, as I saw it  varied among configurations in original samples.